dask_expr/tests/test_format.py

```diff
@@ -95,3 +95,88 @@ def test_series_format():
     H      ..."""
     )
     assert ds.to_string() == exp
+
+
+def test_series_repr():
+    s = pd.Series([1, 2, 3, 4, 5, 6, 7, 8], index=list("ABCDEFGH"))
+    ds = from_pandas(s, 3)
+
+    exp = dedent(
+        """\
+        Empty Dask Series Structure:
```

Comment:
Is the "Empty" a bug here?

Reply:
Thanks for noticing, @jorisvandenbossche! 
Fixed w/ https://github.com/dask/dask-expr/pull/963